lms/static/js/edxnotes/views/note_group.js

```diff
@@ -65,6 +74,6 @@ define([
         }
     });
 
-    return NoteGroupView;
+    return {GroupView: GroupView, ChapterView: CourseStructureChapterView};
```

Comment:
Nit: wouldn't it be clearer to name `CourseStructureChapterView` the same as what it is exported as?